node/src/main/scala/coop/rchain/node/InterpreterRuntime.scala

```diff
@@ -0,0 +1,18 @@
+package coop.rchain.node
+
+import coop.rchain.models.Par
+import coop.rchain.rholang.interpreter.{RholangCLI, _}
+
+object InterpreterRuntime {
+
+  def evaluateFile(fileName: String): Unit = {
+    val persistentStore = RholangCLI.buildStore()
+    val interp          = Reduce.makeInterpreter(persistentStore)
+    val source          = RholangCLI.reader(fileName)
+    RholangCLI
+      .buildNormalizedTerm(source)
+      .foreach((par: Par) => RholangCLI.evaluate(interp, persistentStore, par))
```

Comment:
Why foreach? Has there been some change to the interface?

Reply:
Nope, it was returning `Option`, there was a (brr..... scandal), `get` there ! `foreach` is also not helping :) we should deal with what if it's actually `None`. But I would leave `TODO` there and not block this PR.